.kokoro/build.sh

```diff
@@ -112,6 +112,14 @@ clirr)
     mvn -B -ntp -Denforcer.skip=true clirr:check
     RETURN_CODE=$?
     ;;
+behavioral)
+    # TODO: complete the testing script
```

Comment:
it's not clear to me why this TODO can't be done in this PR?

Reply:
I was a little conservative, I can complete it in one PR.